libursa/src/cl/mod.rs

```diff
@@ -150,7 +150,7 @@ impl CredentialValue {
 }
 
 /// Values of attributes from `Claim Schema` (must be integers).
-#[derive(Debug)]
+#[derive(Debug, Deserialize)]
```

Comment:
Since you are not deriving `Serialize`, how would `CredentialValues` be serialized in the first place?